### PR TITLE
[ADD] 10.0 base_group_permission_information module

### PR DESCRIPTION
**Pull request by damdam-s**
This module allows you to display all permissions informations of a group or a user.

### CODE REVIEW

**Review by nbessi — APPROVED** [review]

File: base_group_permission_information/models/access.py [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+    @api.depends('perm_read', 'perm_write', 'perm_create', 'perm_unlink')
+    def compute_perm_int_vals(self):
+        for res in self:
+            res.update({'read_int': int(res.perm_read),
+                        'write_int': int(res.perm_write),
+                        'create_int': int(res.perm_create),
+                        'unlink_int': int(res.perm_unlink),
+                        })
+
+    create_int = fields.Integer(compute=compute_perm_int_vals, store=True)
```
Comment (simahawk) [inline base_group_permission_information/models/access.py:20]: do we really need these for pivot only?
If mandatory, then add a comment pls

File: base_group_permission_information/models/access.py [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
```
Comment (simahawk) [inline base_group_permission_information/models/access.py:38]: same

File: base_group_permission_information/readme/DESCRIPTION.rst [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+* all inherited/implied groups
+* all rules that applies this group and its implied ones
+* all access control that applies this group and its implied ones
+* all menus accessible from this group and its implied ones
+* all views accessible from this group and its implied ones
+
+User
+----
+
+Displays:
```
Comment (simahawk) [inline base_group_permission_information/readme/DESCRIPTION.rst:17]: a screenshot would be nice

**Review by simahawk — CHANGES_REQUESTED** [review]
also, remove empty files from `readme` folder

**Pull request closed without merging**